apps/wallet-mobile/src/features/Receive/common/CaptureShareQRCodeCard/CaptureShareQRCodeCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useRef } from 'react'
+import { Image, StyleSheet, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+import QRCode from 'react-native-qrcode-svg'
+import { Text } from 'src/components'
+import { COLORS } from 'src/theme'
+
+type ShareProps = {
+    address?: string
+    title?: string
+    addressDetails?: AddressDetailsProps
+}
+
+type AddressDetailsProps = {
+    address: string
+    stakingHash?: string
+    spendingHash?: string
+    title?: string
+}
+
+export function CaptureShareQRCodeCard({ address }: ShareProps) {
+    const LogoRef = useRef(null);
+
+    return (
+        <View
+            style={[styles.touchableCard]}
+        >
+            <LinearGradient
+                style={[StyleSheet.absoluteFill, { opacity: 1 }]}
+                start={{ x: 0, y: 0 }}
+                end={{ x: 0, y: 1 }}
+                colors={['#E4E8F7', '#C6F7F7']}
+            />
+
+            <Image ref={LogoRef.current} source={require('../assets/img/yoroi-logo-text.png')} style={{ height: 40, width: 40 }} />
```

Comment:
We avoid png images. Could you use the svg and transform it to react native component using this tool: https://transform.tools/svg-to-react-native